pelops/experiment_api/experiment.py

```diff
@@ -51,9 +53,11 @@
     -s, --cams=<NUM_CAMS>           : Each camera maps to a set. NUM_CAMS specify the number of camera sets to be outputted.
     -c, --cars=<NUM_CARS_PER_CAM>   : Each set has a list of images. NUM_CARS_PER_CAM specify the number of car images in each camera set.
     -d, --drop=<DROP_PERCENTAGE>    : The likelihood that the target car image is dropped (float from [0,1])
-    -y, --type=<TYPE>               : Determine which type of images to use.
-                                      0: all, 1: query, 2: test, 3: train
     -e, --seed=<SEED>               : Seed to be used for random number generator.
+    -y, --stype=<TYPE>              : Determine which type of images to use.
+                                      0: all, 1: query, 2: test, 3: train
+    -w, --dtype=<DATASET_TYPE>      : Specify the datasets to use. 
```

Comment:
Those are old comments for docopt. They are not exactly how argparse handles it. Argparse just takes in -y or -w. If you preferred the longer flag, let me know.